gulpfile.js

```diff
@@ -4,21 +4,64 @@ var gulp = require('gulp');
 var casperjs = require('gulp-casperjs');
 var spawn = require('child_process').spawn;
 var gutil = require('gulp-util');
+var run = require ('gulp-run');
+var server = require('gulp-develop-server');
+var gulpSequence = require('gulp-sequence');
 
-gulp.task('integrate', function(){
-  gulp.src('./spec/integration/**')
-    .pipe(casperjs({command:'test'}));
+gulp.task('test:integration', function (cb) {
+ console.log('running integration tests');
+
+ var integrationTestGlob = './spec/integration/**';
+
+ gulp.src(integrationTestGlob)
+   .pipe(casperJs({command:'test'}))
```

Comment:
I think this is the misspelling.  JSHint caught this error as well as some style errors around this.  Do you have JSHint enabled in your text editor?